card/src/main/java/com/adyen/checkout/card/internal/ui/DefaultCardDelegate.kt

```diff
@@ -733,6 +730,18 @@ internal class DefaultCardDelegate(
         }
     }
 
+    private fun getCardBrand(detectedCardTypes: List<DetectedCardType>): String? {
+        return if (isDualBrandedFlow(detectedCardTypes)) {
```

Comment:
Minor: instead of calling `?.cardBrand?.txVariant` 3 times you can do it only once after the whole if statement